app/views/spree/admin/log_entries/index.html.erb

```diff
@@ -35,16 +35,17 @@
 
 <%= paginate @log_entries %>
 
+<% if Spree.respond_to?(:solidus_gem_version) && Spree.solidus_gem_version.release >= Gem::Version.new('1.3') %>
```

Comment:
Sure, I was trying to maintain as much parity with the conditional on the other view as possible, since it's a confusing assertion, but not quite worth firing up a view helper over.